phoenicis-library/src/main/java/org/phoenicis/library/ShortcutManager.java

```diff
@@ -79,7 +79,13 @@ public void createShortcut(ShortcutDTO shortcutDTO) {
 
             try {
                 if (shortcutDTO.getIcon() != null) {
-                    File file = new File(shortcutDTO.getIcon());
+                    File file;
```

Comment:
Can we maybe move the `File` creation to a new method?
I mean both changes look equal to me with the difference of two parts, which could be specified using two input parameters.

Reply:
Technically that's of course possible but honestly it doesn't make much sense for me. If you really want it, I can do it.

Reply:
I don't really like this kind of object creation where we begin with `File file;`.
Therefore I think a function would be a more clean solution for this.